app/helpers/sort_helper.rb

```diff
@@ -214,6 +216,12 @@ def sort_clause
     @sort_criteria.to_sql
   end
 
+  # Determines whether the current selected sort criteria
+  # is identical to the default
+  def default_sort_order?
+    @sort_default == @sort_criteria.criteria
```

Comment:
It works as I set  `sort_init` to the full default ascending sort `['lft', true]`, which is effectively the same as calling `from_param('lft'', 'asc')`.